browser/js/common/directives/navbar/navbar.html

```diff
@@ -3,19 +3,21 @@
         <div class="navbar-header">
             <fullstack-logo></fullstack-logo>
         </div>
+
         <ul class="nav navbar-nav">
-            <li ng-repeat="item in items" ng-show="!item.auth || isLoggedIn()">
+            <li ng-repeat="item in items" ng-show="!item.auth || isLoggedIn() && user.firstName !== 'Unregistered'">
```

Comment:
Sam and I came up with this idea, we need to use back end to fix this if we want to dry it up.